src/app/components/generics/tree-output/tree-output.component.scss

```diff
@@ -0,0 +1,13 @@
+
+.tree-wrapper{
+  margin: 16px;
```

Comment:
Boostrap margins should be used if possible

Reply:
I have no idea why this file has been changed. I certainly, at least not by intention, changed anything in it.